packages/editor/src/components/alignment-toolbar/index.js

```diff
@@ -35,19 +35,19 @@ const ALIGNMENT_CONTROLS = [
 	},
 ];
 
-export function AlignmentToolbar( { isCollapsed, value, onChange } ) {
+export function AlignmentToolbar( { isCollapsed, value, onChange, alignmentControls = ALIGNMENT_CONTROLS } ) {
```

Comment:
Should we we add a unit test for this new prop? I see that the component already has some unit tests.

Also, a nit: Renaming the constant to `DEFAULT_ALIGNMENT_CONTROLS` would better convey what it now does.

Reply:
Was thinking we should document this new prop as well. Sadly, it looks like `AlignmentToolbar` is missing a `README`...